### ISSUE TITLE
Keyword error in pysam (via trackster)

### ISSUE THREAD

**Issue by carlfeberhard** [galaxyproject/galaxy#1128]
```
Traceback (most recent call last):
  File "/Users/carleberhard/galaxy/qa/lib/galaxy/datatypes/converters/interval_to_tabix_converter.py", line 40, in 
    main()
  File "/Users/carleberhard/galaxy/qa/lib/galaxy/datatypes/converters/interval_to_tabix_converter.py", line 30, in main
    already_compressed=True, index_filename=out_fname)
  File "pysam/ctabix.pyx", line 738, in pysam.ctabix.tabix_index (pysam/ctabix.c:8259)
TypeError: tabix_index() got an unexpected keyword argument 'index_filename'
```

Steps:
- set up your install for trackster
- use UCSC to get sacCer
- use the visualization button on the dataset to launch trackster in a new visualization
  -> Should show the error above in the 'view error' link

**Comment by dannon** [comment]
@carlfeberhard Thanks.  I have a fix for this one locally that I need to get out, but it involves a new pysam wheel.

**Comment by carlfeberhard** [comment]
Sounds good!

**Comment by yhoogstrate** [comment]
@dannon Are we talking about pysam 0.8.4?

**Comment by dannon** [comment]
@yhoogstrate Yep!  I'm trying to get it out today.

**Comment by dannon** [comment]
Should be resolved with #1638
